### PR TITLE
added ips and email address to mock ds

### PR DESCRIPTION
**Pull request by gsheni**
- Added new Variable Type, IP Address and email address
- Added these types to mock_ds
- Added to docs

### CODE REVIEW

File: docs/source/api_reference.rst [hunk 1]
```diff
@@ -306,6 +306,8 @@ Variable types
     Text
     LatLong
     ZIPCode
+    IP
```
[the remark this answers is not in the document]
Comment (gsheni) [reply docs/source/api_reference.rst]: Fixed.

**Comment by rwedge** [comment]
Thematically, wouldn't IP address make more sense as as session variable and email as a customer variable?

**Comment by gsheni** [comment]
> Thematically, wouldn't IP address make more sense as as session variable and email as a customer variable?

Fixed.

File: featuretools/tests/entityset_tests/test_entity.py [hunk 2]
```diff
@@ -75,10 +75,10 @@ def test_update_data(es):
         es['customers'].update_data(df.drop(columns=['cohort']))
     assert 'Updated dataframe is missing new cohort column' in str(excinfo)
 
-    error_text = 'Updated dataframe contains 13 columns, expecting 12'
+    error_text = 'Updated dataframe contains 14 columns, expecting 13'
```
Comment (gsheni) [inline featuretools/tests/entityset_tests/test_entity.py]: 1 more column(email) was added to the customer's entity.

File: featuretools/tests/entityset_tests/test_entity.py [hunk 3]
```diff
@@ -75,10 +75,10 @@ def test_update_data(es):
...
     with pytest.raises(ValueError, match=error_text) as excinfo:
         es['customers'].update_data(df)
-    assert 'Updated dataframe contains 13 columns, expecting 12' in str(excinfo)
+    assert 'Updated dataframe contains 14 columns, expecting 13' in str(excinfo)
```
Comment (gsheni) [inline featuretools/tests/entityset_tests/test_entity.py]: 1 more column(ip) was added to the session's entity.

**Pull request merged**